### ISSUE TITLE
[Bug]: "facets" as a PIM property should by defined without using XML Schema syntax

### ISSUE THREAD

**Issue by JosefKaltwasser** [DATEX-II-EU/DatexII#519]
### What happened?

"facets" is listed as a Stereotype property for the Platform Independent Model PIM, but it uses XML schema syntax. The defintion in section 6.5.8 should  not not include the "<xs:" prefix and the "/>" postfix, e.g. maxInclusive should be "maxInclusive value='x'" instead of "<xs:maxInclusive value='x'/>.". The wrapping for XML schema should be described in Annex A.3.

### Version

3

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by iancornwellmottmac** [comment]
I think the current approach is as I had envisaged, but I do agree that it could be improved. We needed a platform-independent definition language with semantics; there was no ideal candidate, so we chose XML Schema properties to avoid introducing a new syntax. But since we end up defining all of the keywords and their semantics anyway, rather than referring to XML Schema for their definition, I agree we should drop the outer XML-specific wrapper.
